Test2/Pages/SettingPage.cs

```diff
@@ -0,0 +1,104 @@
+        public IWebElement DefaultCurrency => driver.FindElement(By.XPath("//th[text()='Default currency:']/..//select[@id='currency']"));
+        public IWebElement NumberFormat => driver.FindElement(By.XPath("//th[text()='Number format:']/..//select[@id='numberFormat']"));
+        public IWebElement ButtonSave => driver.FindElement(By.Id("save"));
+        public IWebElement LogutButton => driver.FindElement(By.XPath("//div[contains(@class, 'login')]"));
+        public IWebElement simb => driver.FindElement(By.Id("currency"));
+
+        public string EndDay => driver.FindElement(By.XPath("//th[text()='End Date: *']/..//input")).GetAttribute("value");
+
+        private void WaitForReady()
+        {
+            try
+            {
+                WebDriverWait wait = new WebDriverWait(driver, TimeSpan.FromSeconds(2));
+                wait.Until(ExpectedConditions.ElementToBeClickable(ButtonSave));
+            }
+            catch (Exception ex)
+            {
+            }
+        }
+        private void WaitForAlert()
+        {
+            new WebDriverWait(driver, TimeSpan.FromSeconds(10))
+             .Until(ExpectedConditions.AlertIsPresent());
+            IAlert alert = driver.SwitchTo().Alert();
+            alert.Accept();
+        }
+        public void Сonfigure(string format)
+        {
+            SelectElement dateFormSeletct = new SelectElement(DateFormat);
+            dateFormSeletct.SelectByText(format);
+            WaitForReady();
+            ButtonSave.Click();
+            WaitForAlert();
+            new WebDriverWait(driver, TimeSpan.FromSeconds(10))
+             .Until(ExpectedConditions.TitleContains("Deposite"));
+        }
+        public List<string> Defaultcurrenc
+        {
+            get
+            {
+                List<string> actuale = new List<string>();
+                SelectElement currencySeletct = new SelectElement(DefaultCurrency);
+                foreach (IWebElement element in currencySeletct.Options)
+                {
+                    actuale.Add(element.Text);
+                }
+                return actuale;
+            }
+        }
+        public void Currencysymbol(string curren, string simcur)
+        {
+            SelectElement currencySelect = new SelectElement(DefaultCurrency);
+            currencySelect.SelectByText(curren);
+            WaitForReady();
+            ButtonSave.Click();
+            WaitForAlert();
+            new WebDriverWait(driver, TimeSpan.FromSeconds(10))
+            .Until(ExpectedConditions.UrlContains("Calculator"));
+        }
+        public void Logut()
+        {
+            LogutButton.Click();
+            new WebDriverWait(driver, TimeSpan.FromSeconds(10))
+            .Until(ExpectedConditions.TitleContains("Login"));
+        }
+        public void NumbFormat(string format, string expectedincom, string expectedinterest)
```

Comment:
It should be no 'test' in page-object class, only web-controls and actions of the page. Let's discuss it.